plugins/lighthouse/src/components/Cards/LastLighthouseAuditCard.tsx

```diff
@@ -105,5 +106,12 @@ export const LastLighthouseAuditCard: FC<{ dense?: boolean }> = ({
       <LighthouseAuditSummary audit={website.lastAudit} dense={dense} />
     );
   }
-  return <InfoCard title="Lighthouse Audit">{content}</InfoCard>;
+  return (
+    <InfoCard
+      title="Lighthouse Audit"
+      variant={variant === 'grid-item' ? 'height100' : ''}
```

Comment:
So I'm actually thinking that we at least start out hardcoding it to the `'grid-item'` variant and see if that works out.